src/backend/InvenTree/build/test_build.py

```diff
@@ -722,6 +724,57 @@ def test_new_build_notification(self):
 
         self.assertTrue(messages.filter(user__pk=4).exists())
 
+    def test_events(self):
+        """Test that build events are triggered correctly."""
+
+        from django_q.models import OrmQ
+        from build.events import BuildEvents
+
+        settings.TESTING_TABLE_EVENTS = True
+        settings.PLUGIN_TESTING_EVENTS = True
+        set_global_setting('ENABLE_PLUGINS_EVENTS', True)
```

Comment:
Q: Could a context be used to ensure this setup and teardown are always done correctly?

Reply:
Will do